packages/vuetify/src/composables/border-radius.ts

```diff
@@ -1,15 +1,18 @@
 // Utilities
 import { computed } from 'vue'
+import type { Prop } from 'vue'
```

Comment:
move to line 7

Reply:
K

Reply:
Wait, are you sure you want to mix import from exports?

Also, if we make a separation from what we're importing from public (vue), from internal package, from what we're exporting.  This won't scale well.